src/main/java/it/gov/pagopa/payhub/activities/activity/debtposition/ValidateDebtPositionActivityImpl.java

```diff
@@ -0,0 +1,152 @@
+package it.gov.pagopa.payhub.activities.activity.debtposition;
+
+import it.gov.pagopa.payhub.activities.dao.IngestionFlowFileDao;
+import it.gov.pagopa.payhub.activities.dao.TaxonomyDao;
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.PersonDTO;
+import it.gov.pagopa.payhub.activities.dto.TransferDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.DebtPositionDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.DebtPositionTypeOrgDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.InstallmentDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.PaymentOptionDTO;
+import it.gov.pagopa.payhub.activities.exception.ValidationException;
+import it.gov.pagopa.payhub.activities.utility.Utilities;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static it.gov.pagopa.payhub.activities.utility.Utilities.isValidIban;
+import static it.gov.pagopa.payhub.activities.utility.Utilities.isValidPIVA;
+
+@Lazy
+@Service
+public class ValidateDebtPositionActivityImpl implements ValidateDebtPositionActivity {
+
+    private final IngestionFlowFileDao ingestionFlowFileDao;
+    private final TaxonomyDao taxonomyDao;
```

Comment:
is this used? I cannot see an usage outside the constructor

Reply:
yes, in method "checkTaxonomyCategory"